doc/delivery.md

```diff
@@ -186,7 +186,7 @@ deliveryClient.items<Movie>()
 
 This example returns all **Movie** content items whose **title** element is equal to **Warrior**. Filters are also considered query parameters and can be combined. See [Content filtering](https://developer.kenticocloud.com/v1/reference#content-filtering) in the Kentico Cloud API reference for more general examples.
 
-Supported filters:  `type`,  `types`, `allFilter`, `anyFilter`, `containsFilter`, `equalsFilter`, `greaterThanFilter`, `greaterThanOrEqualFilter`, `infilter`, `lessThanFilter`, `lessThanOrEqualFilter`, `rangeFilter`.
+Filters accepting two parameters:  `type`,  `types`, `equalsFilter`, `greaterThanFilter`, `greaterThanOrEqualFilter`, `lessThanFilter`, `lessThanOrEqualFilter`, `rangeFilter`.
```

Comment:
@kentico-anthonym , I'm not sure I like this distinction. What I would possibly liked is to have something like https://github.com/Enngage/kentico-cloud-js/blob/master/doc/delivery.md#client-configuration - a table which could list all supported params as well as their possible input parameters. It's also possible there are (?) or will be more ways to call these methods and they could easily be incorporated in such a table if necessary.  Would you be able to create something like this?

Reply:
Yeah I'm working on it.